resolwe_bio/tools/summarize_expression_ncRNA.py

```diff
@@ -31,8 +31,8 @@
 
 gene_id_re = re.compile(r'ID=([\w\-\.]*)')
 gene_name_re = re.compile(r'oId=([\w\-\.]*)')
-class_code_re = re.compile(r'class_code=([\w\-\.]*)')
 parent_id_re = re.compile(r'Parent=([\w\-\.]*)')
+class_code_re = re.compile(r'class_code=([\w\-\.]*)')
```

Comment:
There is already pull request for this problem: https://github.com/PyCQA/pycodestyle/pull/593